.github/workflows/repo.yaml

```diff
@@ -1,14 +1,17 @@
-name: repo
+name: Test
+
 on:
   pull_request:
-  push:
     branches:
       - main
+
```

Comment:
Edited this a bit. Do we want a full test suite to run on each merge to main? If we do, we should probably add it to the `publish.yaml` workflow ahead of publishing.

Reply:
Changesets creates a PR with just version bumps right?

Think test CI should run in that branch ahead of merge & publish. I think that would suffice 🤔